src/Faker/Extension/FileExtension.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Faker\Extension;
+
+/**
+ * @experimental This interface is experimental and does not fall under our BC promise
+ */
+interface FileExtension extends Extension
+{
+    /**
+     * Get a random MIME type
+     *
+     * @example 'video/avi'
+     */
+    public function mimeType(): string;
+
+    /**
+     * Get a random file extension (without a dot)
+     *
+     * @example avi
+     */
+    public function extension(): string;
+
+    /**
+     * Get a full path to a new real file on the system.
+     */
+    public function filePath(): string;
```

Comment:
Im doubting if we should keep this in though. Since if km correct it links to the 'file' method as well to generate the actual file?

Reply:
What do you mean? 

My plan is to eventually deprecate `Generator::file()` and replace it with `Generator::filePath()`.